e2e/tests/actionValues.test.ts

```diff
@@ -72,8 +72,12 @@ describe('Assertion and Action values', () => {
     await editAssertion(electronWindow);
     await editAction(electronWindow);
 
+    // test fails without waiting for state to update
+    await new Promise(r => setTimeout(r, 1000));
```

Comment:
```suggestion
  await electronWindow.waitForSelector(`id=action-element-0-0 >> div:has-text("${ACTION_URL}")`, { timeout: 2000 });

```

Maybe we could do something like this: It waits until the element is visible with the given selector. The selector I specified here contains the new url for the action. When the url is updated for the action, this line will resolve and we can go ahead and test the export feature (this is based on the understanding that we can say _the state is updated when the new value is visible in the Action header_)

We can move this line to `editAction` function as well so as it doesn't distract future readers.

** This will fail for now as we have a bug in the main branch that it shows the wrong title in the action title**

Reply:
https://github.com/elastic/synthetics-recorder/pull/212

I applied this change as part of bug fix. I will try to run several tests tomorrow so we can see if it's an effective fix or not.